### ISSUE TITLE
Windows 10 and  Xming- Not loading

### ISSUE THREAD

**Issue by gmalenko** [Microsoft/vscode#13438]
- VSCode Version: code-insider-1.6.0-1475776934_amd64.tar.gz
- OS Version: ubuntu 16.04 and ubuntu 14

Steps to Reproduce:
1. Download tar.gz file and extract.  
2. Run "./code &"
   
   I downloaded the tar.gz inside file (code-insider-1.6.0-1475776934_amd64.tar.gz). I extracted it and ran "./code-insiders"

I got

gmalenko@ubuntu:~/VSCode-linux-x64$ ./code-insiders
bash: cannot set terminal process group (-1): Inappropriate ioctl for device
bash: no job control in this shell
[xcb] Unknown sequence number while processing queue
[xcb] Most likely this is a multi-threaded client and XInitThreads has not been called
[xcb] Aborting, sorry about that.
ome/gmalenko/VSCode-linux-x64/code-insiders: ../../src/xcb_io.c:274: poll_for_event: Assertion `!xcb_xlib_threads_sequence_lost' failed.
Aborted (core dumped)

I'm going to do the deb file and see how it goes

Installed deb file. Run "code-insiders" from the command prompt and nothing really happens. It appears that a process does start

gmalenko@ubuntu:~/VSCode-linux-x64$ ps -aux | grep code
gmalenko 24259 18.0 5.2 866912 53000 ? Ssl 14:46 0:00 /usr/share/cod -insiders/code-insiders
gmalenko 24261 2.5 2.9 309164 30080 ? S 14:46 0:00 /usr/share/cod -insiders/code-insiders --type=zygote --no-sandbox
gmalenko 24265 0.0 0.0 14224 932 pts/0 S+ 14:46 0:00 grep --color=auto code

**Comment by Tyriar** [comment]
Closing as this has likely been fixed.

If not can you clarify what you mean by Windows 10 in the title? Was this in WSL? A VM?

**Comment by gmalenko** [comment]
Unfortunately the issue hasn't been resolved.

Just tried again using "code-insider-1.12.0-1493823801_amd64.tar.gz".   


[xcb] Unknown sequence number while processing queue
[xcb] Most likely this is a multi-threaded client and XInitThreads has not been called
[xcb] Aborting, sorry about that.
ome/gmalenko/vscode/VSCode-linux-x64/code-insiders: ../../src/xcb_io.c:274: poll_for_event: Assertion `!xcb_xlib_threads_sequence_lost' failed.

When I originally made the ticket I was using a factory installed version of Winodws 10 (came with the laptop).  For this test I used Windows Server 2012 R2.  I ran the tar.gz package on a Ubuntu 16.4.2 VM.
